lib/internal/process/promises.js

```diff
@@ -29,6 +38,10 @@ function hasRejectionToWarn() {
 }
 
 function promiseRejectHandler(type, promise, reason) {
+  if (state === undefined) {
+    getOptionValue = require('internal/options').getOptionValue;
```

Comment:
```suggestion
    ({ getOptionValue } = require('internal/options'));
```
I'm wondering why this is memoized at module scope, when this line doesn't check to see that it exists before re-requiring? in other words, why isn't this `const { getOptionValue } = require('internal/options');`?

Reply:
This module is loaded during the execution of `bootstrap/node.js` and we forbid access to CLI options during that time - `getOptionValue` serializes the CLI options to JS land so it needs to be called lazily.

Reply:
I understand why it needs to be called lazily, but not why it needs to be stored in an unused module-level variable.

Reply:
Perhaps this line needs to wrap the lazy require in `if (!getOptionValue)`?

Reply:
It is not an unused module-level variable. If it would be, the linter would complain about it. It is used for [`--abort-on-uncaught-exception`](https://github.com/nodejs/node/pull/26599/files#diff-abbefee1c196cd57720161c414eaaa96R205).

I personally prefer not to use destructuring in combination with brackets here. It's IMHO just less readable.

Reply:
I changed it to use local destructuring now that it's not needed anywhere else in the file anymore.